init.lua

```diff
@@ -82,6 +105,7 @@ local function preview()
 
    -- create a list that holds the clients to preview, from left to right
    local leftRightTab = {}
+   local leftRightTabToAltTabIndex = {} -- save mapping from leftRightTab to altTabTable as well
```

Comment:
Using a second table is not necessary from my point of view, as we can trigger preview activation with a mouse event as you already did a few lines below with the event handler.